src/model/Subscription.ts

```diff
@@ -165,6 +166,11 @@ type _SubscriptionEstimate<IsComplex extends boolean> = {
   continuous_profiling: MaybeComplexFormattedCostWithQuantity<IsComplex>;
   observability_suite: MaybeComplexFormattedCostWithQuantity<IsComplex>;
   resources_total: MaybeComplexFormattedCost<IsComplex>;
+  components?: {
+    [k: string]: MaybeComplexFormattedCostWithTitle<IsComplex>;
```

Comment:
just wondering, what is this exactly this key?

Reply:
two of the keys are known (the next two lines below), but this object can contain arbitrary others. we know the keys will be strings and also the shape of the values, and can map over them to list billing line items.